packages/aws-cdk/bin/cdk.ts

```diff
@@ -318,7 +318,7 @@ async function initCommandLine() {
 
     // not outputting template to stdout, let's explain things to the user a little bit...
     success(`Successfully synthesized to ${colors.blue(path.resolve(appStacks.assembly!.directory))}`);
-    print(`Supply a stack name (${stacks.map(s => colors.green(s.name)).join(', ')}) to display its template.`);
+    print(`Supply a stack id (${stacks.map(s => colors.green(s.id)).join(', ')}) to display its template.`);
```

Comment:
This could potentially be a problem. Will users KNOW the stack ID? Do we show it in `cdk ls`? If we do, we should show all of ID, name and environment because if not it's going to get confusing.

Reply:
We IDs in `cdk ls` and both in `cdk ls --long`